osmosis-installer.py

```diff
@@ -5,17 +5,63 @@
 import readline
 import random
 import argparse
+import sys
 from os import remove
-from sys import argv
 
-remove(argv[0])
-
-parser = argparse.ArgumentParser(description="Use default settings")
-parser.add_argument('-m', action='store_true', help='use default settings with no input for mainnet')
-parser.add_argument('-t', action='store_true', help='use default settings with no input for mainnet')
 
+#remove(argv[0])

```

Comment:
what is this comment?

Reply:
Thanks for catching, this should actually be uncommented. Its what "self-destructs" the python file. I comment it out while testing so it doesn't delete itself.